project1/styles.css

```diff
@@ -0,0 +1,44 @@
+h1,
+h2,
+h3 {
+    text-align: center;
+    background-color: #eb3f33;
+    margin-top: 0;
+    margin-bottom: 0;
+    padding-bottom: 12px;
+}
+
+h1 {
+    font-family: 'Playfair Display', serif;
+}
+
+h2 {
+    font-size: 15px;
+}
+
+h3 {
+    font-family: 'Titillium Web', sans-serif;
+    font-weight: 400px;
+}
+
+body {
+    font-family: 'Nunito Sans', sans-serif;
+    font-size: 18px;
+    font-weight: 0.5;
```

Comment:
This rule is invalid -- the values for `font` weight are either keywords (like `normal` or `bold`) or a number in the hundreds (like `100` or `200`) ranging from 100-900.